docs/documentation/schedule/components/frequency_based_services.md

```diff
@@ -1,8 +1,10 @@
-# Frequency-based services Component & Feature
+# Frequency-based services
 
 <div class="grid" markdown>
 
-The frequencies component and feature can be used to represent frequency-based trips (e.g. a bus trips running every 10 minutes, a subway or metro service running every 2 minutes), this can be a useful alternative to providing specific timetables, acting as an alternative to some of the information included in the stop times Feature of the Base Component. 
+Frequency-based services can be used to model services that operate on a regular frequency, such as buses running every 10 minutes or subway services operating 2 minutes within specified time intervals.
+When modeling a Frequency-based service, `stop_times.txt` is still defined and contains the relative times between stops in order to determine the times to be displayed to riders. This modeling only works in the case where the relative times between stops remains consistent for all stops.
```

Comment:
I think this might be too specific/into-the-weeds, maybe we can simplify it to make sure it references stop_times.txt but without the extra detail.

Reply:
fixed in last commit